utils/vara-ui/src/components/button/button.tsx

```diff
@@ -1,12 +1,13 @@
 import { ButtonHTMLAttributes, FunctionComponent, ReactNode, SVGProps, forwardRef } from 'react';
 import cx from 'clsx';
 import styles from './button.module.scss';
+import type { IButtonColors, IButtonSizes } from './helpers.ts';
 
 type BaseProps = ButtonHTMLAttributes<HTMLButtonElement> & {
   text?: string;
   icon?: FunctionComponent<SVGProps<SVGSVGElement> & { title?: string | undefined }>;
-  color?: 'primary' | 'dark' | 'light' | 'grey' | 'border' | 'transparent';
```

Comment:
There are no more "dark" and "light" color names in the design system.

Since the introduction of the dark theme, these names have become misleading: when working with the dark theme, a "dark" button will appear light.

We can keep them marked as "@deprecated" and add an "additional" color (which matches "dark" in terms of styles), as seen in the design.

Ideally, once the migration is complete we can remove them entirely.
There aren’t many places using them in dapps

Reply:
Extremely good point!